docs/rules/self-closing-comp.md

```diff
@@ -1,4 +1,4 @@
-# Prevent extra closing tags for components without children (self-closing-comp)
+# Prevent extra closing tags for components without children (react/self-closing-comp)
```

Comment:
I think you may have missed some other lines like this one (e.g. see require-optimization). It might be worth making another pass.